projects/packages/my-jetpack/src/class-rest-product-data.php

```diff
@@ -64,7 +64,6 @@ public static function get_all_product_data() {
 
 		// If site has backups and the realtime backup capability, add latest undo event
```

Comment:
Unrelated to this change, should the error check for the API response come before this additional populating of the response body?

Reply:
That's a good call, I think we could move that above